### ISSUE TITLE
Broken Hyperlink on Aframe website - videosphere section

### ISSUE THREAD

**Issue by jadegamesuk** [aframevr/aframe#5021]
Broken hyperlink on your Docs page. 
- A-Frame Version:1.3.0
- Platform / Device:
- Reproducible Code Snippet or URL: https://aframe.io/docs/1.3.0/primitives/a-videosphere.html#attributes - click on the common attributes link and you'll see it doesn't link anywhere. Please fix this.

**Comment by msub2** [comment]
I don't believe this page ever existed. Even if you browse the repo [in the state it was at when that line was first added](https://github.com/aframevr/aframe/blob/b3837ddd93ef9bd1ddf21a70392a63b99fb1cbd8/docs/primitives/a-videosphere.md) it's not there.

**Comment by dmarcos** [comment]
PR welcome to correct.

**Comment by vincentfretin** [comment]
That link was removed, it doesn't exist in https://aframe.io/docs/1.7.0/primitives/a-videosphere.html#attributes
You can close.
